.travis.yml

```diff
@@ -5,9 +5,13 @@ node_js:
   - '6'
   - '4'
   - '0.12'
-
+cache:
+  directories:
+    - node_modules
+before_install:
+- npm update
+- npm prune
```

Comment:
Just to save some more time, you should `prune` _before_ update, otherwise it will spend time updating things you are about to remove in `prune`.